docs/content/latest/deploy/public-clouds/aws/cloudformation.md

```diff
@@ -10,15 +12,18 @@ $ wget https://raw.githubusercontent.com/yugabyte/aws-cloudformation/master/yuga
 Create CloudFormation template:
 
 ```sh
-$ aws cloudformation create-stack --stack-name <stack-name> --template-body yugabyte_cloudformation.yaml --parameters DBVersion=1.2.8.0, KeyName=<ssh-key-name>
+$ aws --region <aws-region> cloudformation create-stack --stack-name <stack-name> --template-body file://yugabyte_cloudformation.yaml --parameters  ParameterKey=DBVersion,ParameterValue=2.0.6.0,ParameterKey=KeyName,ParameterValue=<ssh-key-name>
 ```
 Once resource creation completes, check that stack was created:
 
 ```sh
-$ aws cloudformation describe-stacks --stack-name <stack-name>
+$ aws --region <aws-region> cloudformation describe-stacks --stack-name <stack-name>
 ```
 From this output, you will be able to get the VPC id and YugabyteDB admin URL.
 
+Keep in mind the stack creates a security group restricts access to the DB, you should update the security group inbound rules
```

Comment:
```suggestion
Because the stack creates a security group that restricts access to the database, you should update the security group inbound rules.
```